examples/example-gateway/build/gen-code/clients/contacts/contacts/types_i64.go

```diff
@@ -0,0 +1 @@
+package contacts
```

Comment:
It looks like we generate unneeded empty `types_i64.go` files. We want to somehow not do that.

Reply:
sounds good